src/services/development.ts

```diff
@@ -80,44 +81,29 @@ export const services: ServiceDefinition[] = [
           console.warn('Found an unhandled %s request to %s', req.method, req.url.href)
         },
       })
+      return rest
     },
     arguments: [
       $.mswHandlers,
     ],
   }],
-
-  // old style kuma mocks
-  [$.mswKumaMocks, {
-    service: (env: Env) => setupHandlers(env.var('KUMA_API_URL'), mocks, env),
+  [$.mswFakeApiHandlers, {
+    service: (env: TokenType<typeof $.env>, fs: FS) => fakeApi(env, rest, fs),
     arguments: [
-      $.Env,
+      $.env,
+      $.fakeFS,
     ],
     labels: [
       $.mswHandlers,
     ],
   }],
 
-  // new style kuma mocks
   [$.kumaFS, {
     constant: fs,
-    arguments: [
-      $.Env,
-    ],
     labels: [
       $.fakeFS,
     ],
   }],
-  // new style kuma mocks
-  [$.mswKumaHandlers, {
-    service: (env: TokenType<typeof $.env>, fs: FS) => fakeApi(env, fs),
-    arguments: [
-      $.env,
-      $.fakeFS,
-    ],
-    labels: [
-      $.mswHandlers,
-    ],
-  }],
```

Comment:
This was pretty much just moved, token name changed, and I also added the extra `rest` argument (`fakeApi(env, rest, fs)`). You'll see the final result over in the green of the changeset.